env_config/env.docker

```diff
@@ -44,8 +44,8 @@ DB_SCHEMA=invasivesbc
 # ------------------------------------------------------------------------------
 # KeyClock URLS
 # ------------------------------------------------------------------------------
-APP_CERTIFICATE_URL=https://dev.oidc.gov.bc.ca/auth/realms/dfmlcg7z/protocol/openid-connect/certs
-APP_CERTIFICATE_URL_TEST=https://oidc.gov.bc.ca/auth/realms/dfmlcg7z/protocol/openid-connect/certs
+APP_CERTIFICATE_URL=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs
+APP_CERTIFICATE_URL_TEST=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs
```

Comment:
Old value for `APP_CERTIFICATE_URL_TEST` points to Silver PROD. Is this supposed to point at Gold PROD as well?